app/enterprise/0.35-x/kong-manager/administration/rbac/new-role.md

```diff
@@ -1,40 +1,51 @@
 ---
 title: How to Create a New Role with Custom Permissions
 book: admin_gui
+toc: false
 ---
-**Prerequisites**:
+#### Prerequisites
 
-* Authentication and RBAC are enabled, following the [Getting Started](/enterprise/{{page.kong_version}}/getting-started/start-kong/#prerequisites) guide
-* [Logged in as the Super Admin](/enterprise/{{page.kong_version}}/getting-started/start-kong/#step-4) or a user that has `/admins` and `/rbac` read and write access
-
-<video width="100%" autoplay loop controls>
- <source src="https://konghq.com/wp-content/uploads/2019/02/role-creation-ent-34.mov" type="video/mp4">
- Your browser does not support the video tag.
-</video>
+* Authentication and RBAC are enabled, following the 
+[Getting Started](/enterprise/{{page.kong_version}}/getting-started/start-kong/#prerequisites) 
+guide
+* [Logged in as the Super Admin](/enterprise/{{page.kong_version}}/getting-started/start-kong/#step-4) 
+or a user that has `/admins` and `/rbac` read and write access.
 
+## Create a New Role with Custom Permissions
 
-1. On the "Admins" page, to create a new Role, click the "Add Role" button at the top right of the list of Roles. 
+1. On the "Admins" page, to create a new Role, click the "Add Role" button at 
```

Comment:
Do we want this to match the format of how-to guides mentioned in the technical writing guide, i.e. to have step numbers are anchored headers?

Either way:

```suggestion
1. On the **Admins** page, to create a new **Role**, click the **Add Role** button at 
```